fdbserver/MockGlobalState.actor.cpp

```diff
@@ -379,11 +379,18 @@ Future<Void> MockStorageServer::waitMetricsTenantAware(const WaitMetricsRequest&
 }
 
 void MockStorageServer::getStorageMetrics(const GetStorageMetricsRequest& req) {
-	StorageBytes storageBytes(
-	    totalDiskSpace - usedDiskSpace, totalDiskSpace, usedDiskSpace, totalDiskSpace - usedDiskSpace);
-	metrics.getStorageMetrics(
-	    req, storageBytes, counters.bytesInput.getRate(), 0, now(), 0, counters.bytesInput.getValue());
-	// FIXME: MockStorageServer does not support bytesDurable yet
+	GetStorageMetricsReply reply;
```

Comment:
Why do we make this change?

Reply:
Let me restore the old format (vestige of my many messings w/ metrics)